js-libs/telemetry/TelemetryV3Manager.js

```diff
@@ -117,6 +117,8 @@ TelemetryV3Manager = Class.extend({
             var v3questionItem = {
                 id: eventObj.event.edata.eks.qid,
                 maxscore: eventObj.event.edata.eks.maxscore,
+                eventVer: data.assessEventVer,
```

Comment:
Send eventVer: data.assessEventVer in the second parameter of the EkTelemetry.assess(v3questionData, options);
var options: = {eventVer: data.assessEventVer}